repub.go

```diff
@@ -139,17 +113,20 @@ func (rp *Republisher) Run() {
 
 // Wrapper function around the user-defined `pubfunc`. It publishes
 // the (last) `valueToPublish` set and registers it in `lastValuePublished`.
-func (rp *Republisher) publish(ctx context.Context) error {
+// TODO: Allow passing a value to `PublishNow` which supersedes the
+// internal `valueToPublish`.
```

Comment:
I'm not sure if we want to allow this. Users shouldn't swap out the MFS root using the republisher.

Reply:
I'm not sure I understand the comment, what do you mean by swap out?

The `TODO` (that I don't think I worded correctly) was aiming at adding an optional argument that would replace the `Update(newCid); PublishNow();` call pair with just a `PublishNow(newCid)` call.

Reply:
Got it. Yeah, that makes sense.

(context: I keep thinking we're exposing the republisher to the user)